### PR TITLE
add script for importing localgroups

### PR DESCRIPTION
**Pull request by s-cheng**
I created and used this to import our data from the recent groups census. Probably won't be useful until the next census.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1203926057101708) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/server/scripts/importLocalgroups.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        if (website && !website.startsWith('http')) {
+          website = `https://${website}`
+        }
+        
+        const dataToSet = {
+          name: data.name,
+          nameInAnotherLanguage: data.nameInAnotherLanguage || undefined,
+          inactive: data.inactive === 'TRUE',
+          categories: data.categories.split('; ').map(label => {
+            return GROUP_CATEGORIES.find(cat => cat.label === label)?.value
```
Comment (jpaddison3) [inline packages/lesswrong/server/scripts/importLocalgroups.ts:37]: 🐈

File: packages/lesswrong/server/scripts/importLocalgroups.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        
+        // add the group if we haven't found them
+        if (!data._id) {
+          //eslint-disable-next-line no-console
+          console.log(`${groupName} not found in the db - adding`)
+          const newGroup = await createMutator({
+            collection: Localgroups,
+            document: {
+              ...dataToSet,
+              types: ["LW"], // not used by EA Forum but still required
```
Comment (jpaddison3) [inline packages/lesswrong/server/scripts/importLocalgroups.ts:59]: Why not put `types: ["EA"]` idk literally doesn't matter

File: packages/lesswrong/server/scripts/importLocalgroups.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+          //eslint-disable-next-line no-console
+          console.log(`${newGroup.data._id} - ${groupName} added`)
+        }
+        // replace any existing data with the imported data
+        else {
+          void updateMutator({
+            collection: Localgroups,
+            documentId: data._id,
+            set: dataToSet,
+            unset: {},
```
Comment (jpaddison3) [inline packages/lesswrong/server/scripts/importLocalgroups.ts:73]:
```suggestion
```

[nit] unset is optional, right?

**Review by jpaddison3 — APPROVED** [review]
LGTM

File: packages/lesswrong/server/scripts/importLocalgroups.ts (hunk 2 above)
Comment (s-cheng) [reply packages/lesswrong/server/scripts/importLocalgroups.ts:59]: I believe this is the default value, so just went with that

File: packages/lesswrong/server/scripts/importLocalgroups.ts (hunk 3 above)
Comment (s-cheng) [reply packages/lesswrong/server/scripts/importLocalgroups.ts:73]: Oh I wasn't sure, but looks like it

**Pull request merged**